### DISCUSSION TITLE
'Revive' Mod

### DISCUSSION THREAD

**Discussion by Axeanz** [ppy/osu#13061] [Ideas]
Basically NoFail but with a limit, or Easy without the difficulty-decrease.

You can use Revive with Sudden_Death and Perfect, which will allow you to miss 'x' times or hit 'x' 100s, depending on the amount of revives (max of 3 seems reasonable).
Should probably be Unranked.

Or maybe have something like a setting on sudden_death/perfect, giving you an option on how many misses/100s/50s are allowed, just to get that 'nothing less, only more', and not the 'SS or nothing'

**Comment by holly-hacker** [comment]
Perhaps this could be implemented as a setting of the easy and sudden death mods.

**Comment by frenzibyte** [comment]
Referencing #6272 for usage. (Going with the setting way)
